README.md

```diff
@@ -2,9 +2,14 @@
 Automates the process of making tennis court reservations for you.
 
 ## Usage
-1. Open `AutoTenez.py` and enter your personal details, and desired time slot(s) to play tennis for tomorrow.
-2. Then run `python3 AutoTenez.py` in a terminal to make the reservation.
+1. Open `AutoTenez.py` and enter your personal details.
+1. Run `python3 AutoTenez.py -h` in a terminal to see all options possible. 
```

Comment:
List numbering is incorrect. Will probably be resolved after rebasing and merging #4

Reply:
This is how to do numbering in markdown. Markdown will fix the numbering when parsing this.